src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -114,10 +114,11 @@ export default class BottomNavigationButton extends Component {
       [classes.selected]: selected,
       [classes.selectedIconOnly]: !showLabelProp && !selected,
     }, classNameProp);
-    const iconClassName = classNames(classes.icon,
-      isValidElement(iconProp) ? iconProp.props.className : null);
+    const iconClassName = classNames(classes.icon, isValidElement(iconProp) ?
+      iconProp.props.className :
```

Comment:
It fixes the layout issue - `display: block` doesn't work when applied directly to the SVG icon, but it does when applied to wrapper div. I don't know why. Like I said above: 

> I had a crack at fixing the BottomNav layout, but let me know if there's a better way.